tools/autossh.sh

```diff
@@ -16,7 +16,7 @@ ALIVE=0
 HISTFILE="$HOME/.autossh.history"
 
 # Use colors, but only if connected to a terminal, and that terminal supports them.
-if which tput >/dev/null 2>&1; then
+if _omb_util_command_exists tput; then
```

Comment:
`_omb_util_command_exists` => `_omb_util_binary_exists`

Reply:
Ah, I think we should not change this because `autossh.sh` is not a part of the interactive configuration, so it is executed in a context where `_omb_util_command_exists` is not defined.